src/cryptography/x509/name.py

```diff
@@ -114,11 +114,19 @@ def sub(m):
     return _RFC4514NameParser._PAIR_RE.sub(sub, val)
 
 
-class NameAttribute:
+NameAttributeValueType = typing.TypeVar(
+    "NameAttributeValueType",
+    typing.Union[str, bytes],
+    str,
+    bytes,
+)
```

Comment:
Should this be `covariant=True`?

Reply:
From my understanding yes, updated in 2603a49